docs/admin/federation-apiserver.md

```diff
@@ -1,8 +1,3 @@
 This file is autogenerated, but we've stopped checking such files into the
 repository to reduce the need for rebases. Please run hack/generate-docs.sh to
 populate this file.
```

Comment:
It was just part of auto-generation, i think I am to ok to delete these from this PR.